paddle/fluid/framework/details/nccl_all_reduce_op_handle.cc

```diff
@@ -73,8 +73,9 @@ void NCCLAllReduceOpHandle::RunImpl() {
 
     for (size_t i = 0; i < local_scopes_.size(); ++i) {
       auto *s = local_scopes_[i];
+      auto &local_scope = *s->FindVar(kLocalExecScopeName)->Get<Scope *>();
```

Comment:
Please make the names of different levels' scopes clear in further PRs. The 'local' of line 75 and line 76 means different ...

I think `scope --> local_scope --> exe_scope` is a good way to distinguish scopes of differnet levels.